tools/api-markdown-documenter/package.json

```diff
@@ -14,12 +14,13 @@
 	"main": "dist/index.js",
 	"types": "dist/index.d.ts",
 	"scripts": {
-		"build": "concurrently npm:build:compile npm:lint && npm run build:docs",
+		"build": "concurrently npm:build:compile npm:lint && npm run build:docs npm:build:readme ",
```

Comment:
@tylerbutler presumably this is okay for independent packages. But if we would prefer to keep it separate and add a separate verification step to the pipeline, I can make that change.

Reply:
This should be fine.